src/components/promotion/PromotionDetails.tsx

```diff
@@ -24,15 +23,17 @@ export default function PromotionDetails({
   description,
   expirationDate,
   restaurantName = 'Restaurant',
+  boldName = undefined,
+  boldDescription = undefined,
 }: Promotion): ReactElement {
   return (
     <div style={styles.descriptionContainer}>
       <div style={styles.header}>
-        <h2>{name}</h2>
+        {boldName !== undefined ? parse(boldName) : <h2>{name}</h2>}
         <HeartOutlined style={styles.heart} />
       </div>
       <h3>{restaurantName}</h3>
-      <p>{description}</p>
+      {boldDescription !== undefined ? parse(boldDescription) : <p>{description}</p>}
```

Comment:
And then here you can probably just do `boldName ? parse(boldName) : <h2>{name}</h2>` and likewise for description

Reply:
Ohman idek what I was doing, good point thank you